package.json

```diff
@@ -42,6 +43,8 @@
     "recursive-readdir": "^2.2.3",
     "underscore": "^1.13",
     "undici": "^5.11",
-    "@trivago/prettier-plugin-sort-imports/@babel/parser": "^7.22.7"
+    "@trivago/prettier-plugin-sort-imports/@babel/parser": "^7.22.7",
+    "@typechain/ethers-v5": "11.1.2",
+    "typechain@npm:^8.0.0": "patch:typechain@npm%3A8.3.2#~/.yarn/patches/typechain-npm-8.3.2-b02e27439e.patch"
```

Comment:
can this be devDependency?

Reply:
It is indeed a dev dep of the solidity and helloworld packages. This section here is for resolutions to ensure the patched version gets selected. Yarn added it automatically during the path process